scala3doc/src/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -203,6 +205,31 @@ trait ClassLikeSupport:
         inherited.flatMap(s => parseInheritedMember(s))
     }
 
+    /** Extracts members while taking Dotty logic for patching the stdlib into account. */
+    def extractPatchedMembers: Seq[Member] = {
+      val ownMembers = c.extractMembers
+      def extractPatchMembers(sym: Symbol) = {
```

Comment:
Shouldn't that logic be placed into TASY? I mean  people may wan to parse tasty from stdlib and they will be really supprised what they will get

Reply:
Tasty Reflect is already aware of how the compiler patches stdlib definitions. Our problem is that we rely on class bodies to document classes, and those aren't patched.

We can open a separate issue for patching class bodies as well.